backend/LearningCalendar/Epicenter.Service/Operations/Authentication/Invite/GetInvitationDetailsOperation.cs

```diff
@@ -1,28 +1,46 @@
 ﻿using System.Threading.Tasks;
 using Epicenter.Persistence.Interface.Repository.Authentication;
+using Epicenter.Persistence.Interface.Repository.LearningCalendar;
 using Epicenter.Service.Interface.Operations.Authentication.Invite;
 
 namespace Epicenter.Service.Operations.Authentication.Invite
 {
     public class GetInvitationDetailsOperation : IGetInvitationDetailsOperation
     {
         private readonly IInvitationRepository _invitationRepository;
+        private readonly IEmployeeRepository _employeeRepository;
 
-        public GetInvitationDetailsOperation(IInvitationRepository invitationRepository)
+        public GetInvitationDetailsOperation(
+            IInvitationRepository invitationRepository, 
+            IEmployeeRepository employeeRepository)
         {
             _invitationRepository = invitationRepository;
+            _employeeRepository = employeeRepository;
         }
 
         public async Task<GetInvitationDetailsOperationResponse> Execute(GetInvitationDetailsOperationRequest request)
         {
-
             var invitation = await _invitationRepository.GetWithInviterAsync(request.Id);
 
+            var inviter = await _employeeRepository.GetByIdentityId(invitation.InvitationFromId);

```

Comment:
The method above makes me believe that you should already be able to reach the inviter through the invitation so either naming is wrong or getting the inviter was not necessary

Reply:
It includes the identity user, not the employee